pkg/ansible/roles/bigchaindb-driver/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+dependencies_deb:
+  - python3-dev
+  - libffi-dev
+  - libssl-dev
+  - python3-pip
+
+dependencies_yum:
+  - gcc-c++
+  - "@Development Tools"
+  - python34-devel
```

Comment:
I think we only support python 3.5 and 3.6 now. Refer to https://github.com/bigchaindb/bigchaindb/blob/master/.travis.yml